projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-grid.component.html

```diff
@@ -34,7 +34,7 @@
         | pivotGridSort:pivotConfiguration:sortStrategy:id:pipeTrigger
         | pivotGridRow:pivotConfiguration:expansionStates:pipeTrigger:sortingExpressions
         | pivotGridColumn:pivotConfiguration:expansionStates:pipeTrigger:sortingExpressions
-        | pivotGridColumnSort:sortingExpressions:sortStrategy:pipeTrigger
+        | pivotGridColumnSort:sortingExpressions:sortStrategy:pipeTrigger:pivotConfiguration.pivotKeys
```

Comment:
IMO `pivotConfiguration.pivotKeys` is not always defined. This should be `this.pivotKeys`.